webapp/src/main/java/com/box/l10n/mojito/service/ai/openai/OpenAILLMService.java

```diff
@@ -93,6 +117,54 @@ public AICheckResponse executeAIChecks(AICheckRequest aiCheckRequest) {
     return aiCheckResponse;
   }
 
+  @Override
+  @Timed("OpenAILLMService.translate")
+  public String translate(
+      TMTextUnit tmTextUnit, String sourceBcp47Tag, String targetBcp47Tag, AIPrompt prompt) {
```

Comment:
Could we also track with a metric successful vs. failed requests, to get an idea of the failure % of OpenAI API requests? (ideally, retryable failures should be tracked differently than terminal/non-retryable failure - to get two different computed metrics there)